src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,108 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    if (!value) {
+      return;
+    }
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  handleSubmit = (event, newMovie) => {
+    event.preventDefault();
+
+    const values = Object.values(newMovie);
+
+    if (values.some(value => !value)) {
+      return;
+    }
+
+    this.props.onAdd(newMovie);
+    this.setState({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+  }
 
   render() {
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
+    const newMovie = {
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    };
+
     return (
-      <form>
-        Put the form here
+      <form
+        className="form"
+        onSubmit={(event, movieToAdd) => this.handleSubmit(event, newMovie)}
```

Comment:
just pass event into handleSubmit
```suggestion
        onSubmit={this.handleSubmit}
```